lib/sisense/elasticube_datasecurity.rb

```diff
@@ -3,8 +3,10 @@ class ElasticubeDatasecurity < API::NestedResource
     RESOURCE_NAME = 'datasecurity'.freeze
     PARENT_CLASS = Sisense::Elasticube
 
-    def self.list(server: 'LocalHost', elasticube_title:)
-      path = [resource_base_path(use_legacy_api: true), server, elasticube_title, 'datasecurity'].join('/')
+    def self.list(params: {})
```

Comment:
If some params are required, better to make them explicitly named, as :server and :elasticube_title are. Otherwise, splat the params (**params) and pass them on to whatever is using them.